CMakeLists.txt

```diff
@@ -198,40 +211,40 @@ endif()
 #               QGroundControl Target
 #######################################################
 
-qt_add_executable(${PROJECT_NAME} src/main.cc ${QGC_RESOURCES})
-
-add_subdirectory(libs)
-add_subdirectory(src)
-
-set_target_properties(${PROJECT_NAME}
-    PROPERTIES
-        QT_RESOURCE_PREFIX "/qgc"
+qt_add_executable(${PROJECT_NAME}
+    src/main.cc
+    ${QGC_RESOURCES}
 )
 
-target_precompile_headers(${PROJECT_NAME} PRIVATE src/pch.h)
-
 if(Qt6LinguistTools_FOUND)
     file(GLOB TS_SOURCES RELATIVE ${CMAKE_SOURCE_DIR} ${CMAKE_SOURCE_DIR}/translations/qgc_*.ts)
-    # TODO: Update to new qt_add_translations form in Qt6.7
-    qt_add_translations(${PROJECT_NAME}
-        TS_FILES
-            ${TS_SOURCES}
-    )
+    qt_add_translations(${PROJECT_NAME} TS_FILES ${TS_SOURCES}) # TODO: Update to new qt_add_translations form in Qt6.7
 endif()
 
-if(MACOS)
+target_precompile_headers(${PROJECT_NAME} PRIVATE src/pch.h)
+
+set_target_properties(${PROJECT_NAME} PROPERTIES QT_RESOURCE_PREFIX "/qgc")
+
+if(WIN32)
+    set_target_properties(${PROJECT_NAME} PROPERTIES WIN32_EXECUTABLE TRUE)
+    add_compile_definitions(_USE_MATH_DEFINES)
+elseif(MACOS)
     set_target_properties(${PROJECT_NAME}
         PROPERTIES
             MACOSX_BUNDLE TRUE
             MACOSX_BUNDLE_INFO_PLIST ${CMAKE_SOURCE_DIR}/deploy/mac/MacOSXBundleInfo.plist.in
+            MACOSX_BUNDLE_BUNDLE_NAME ${PROJECT_NAME}
+            MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION}
+            MACOSX_BUNDLE_COPYRIGHT "Copyright (c) 2018 QGroundControl. All rights reserved."
+            MACOSX_BUNDLE_GUI_IDENTIFIER "io.mavlink.qgroundcontrol"
+            MACOSX_BUNDLE_ICON_FILE "${CMAKE_SOURCE_DIR}/resources/icons/macx.icns"
```

Comment:
Are these the additions you are talking?

Reply:
Yeah, they were taken from [here](https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#properties-on-targets).